ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -180,16 +413,62 @@ const triggerSet: TriggerSet<Data> = {
       netRegexDe: NetRegexes.startsUsing({ id: '69D6', source: 'Hesperos', capture: false }),
       netRegexFr: NetRegexes.startsUsing({ id: '69D6', source: 'Hespéros', capture: false }),
       netRegexJa: NetRegexes.startsUsing({ id: '69D6', source: 'ヘスペロス', capture: false }),
-      infoText: (_data, _matches, output) => output.text!(),
+      preRun: (data) => data.pinaxCount = (data.pinaxCount ?? 0) + 1,
+      infoText: (data, _matches, output) => {
+        if ((data.pinaxCount ?? 0) % 2)
+          return output.text!();
+        data.wellShiftKnockback = true;
+        return output.shiftWell!();
+      },
       outputStrings: {
         text: {
           en: 'Well Pinax',
           de: 'Brunnen-Pinax',
           fr: 'Pinax d\'eau',
+          ja: '水ピナクス',
```

Comment:
As a general rule, I think it's better to not merge in translations until after a PR has landed.  It makes it much harder for me to read.

Reply:
I'll keep that in mind next time I see a PR on my branch.